### PR TITLE
Added intensifiers functionality

### PR DESCRIPTION
**Pull request by edmondlafay**
I was frustrated by having a positive score for 'I really hate ruby', so I added a dictionary with intensifier words that fixes this case.
I left the words in the scoring dictionary so to have scores even for a one word text (like a tweet that responds to a question : 'Are you happy?' -> 'Very'

### CODE REVIEW

**Comment by Skizzk** [comment]
Can we have some tests please ?

File: lib/sentimental.rb [hunk 1]
```diff
@@ -44,6 +66,17 @@ def load_senti_file(filename)
     end
   end
 
+  def load_multipliers_file(filename)
```
Comment (Skizzk) [inline lib/sentimental.rb]: I think that this can be refactorized with load_senti_file

**Comment by edmondlafay** [comment]
I see Skizzk made a pull request that is much better than mine and compliant with most text analysis reaserch, but is incompatible with this pull request. I will therefore delete this one.

**Comment by Skizzk** [comment]
I can add/code your idea if you want. It's a good one.

**Comment by edmondlafay** [comment]
Thank you for your support, but I believe that your n-grams implementation is a much better idea since generating dictionaries for it can be automated, whereas my approche requires a hand made list.
Plus it is what the great majority of text analysers do, so I think it will be easier to build on top of your code than mine.

**Comment by Skizzk** [comment]
Ok. I might need it anyway. It will be my next commit ;) (in a week or so)

**Comment by Skizzk** [comment]
Finally done it. Tell me if it is ok for you in that form. Sorry for the delay >< (busy times)

**Comment by edmondlafay** [comment]
Nice work, very smart.

**Pull request closed without merging**